pkg/registration/registration_client.go

```diff
@@ -161,10 +169,35 @@ func (rClient *K8sRegistrationClient) addActionPolicy(ab *builder.ActionPolicyBu
 	}
 }
 
-func (rClient *K8sRegistrationClient) GetEntityMetadata() []*proto.EntityIdentityMetadata {
-	glog.V(3).Infof("Begin to build EntityIdentityMetadata")
+func (rClient *K8sRegistrationClient) GetActionMergePolicy() []*proto.ActionMergePolicyDTO {
+	glog.V(2).Infof("Begin to build Action Merge Policies")
+
+	actionMergeTarget := builder.NewActionDeDuplicateAndAggregationTargetBuilder().
+		DeDuplicatedBy(builder.NewActionAggregationTargetBuilder(proto.EntityDTO_CONTAINER_SPEC,
+			proto.ConnectedEntity_AGGREGATED_BY_CONNECTION)).
+		AggregatedBy(builder.NewActionAggregationTargetBuilder(proto.EntityDTO_WORKLOAD_CONTROLLER,
+			proto.ConnectedEntity_CONTROLLED_BY_CONNECTION))
+
+	actionMergeTarget2 := builder.NewActionDeDuplicateAndAggregationTargetBuilder().
+		DeDuplicatedBy(builder.NewActionAggregationTargetBuilder(proto.EntityDTO_CONTAINER_SPEC,
+			proto.ConnectedEntity_AGGREGATED_BY_CONNECTION)).
+		AggregatedBy(builder.NewActionAggregationTargetBuilder(proto.EntityDTO_WORKLOAD_CONTROLLER,
+			proto.ConnectedEntity_OWNS_CONNECTION))
+
+	containerResizeMerge := builder.NewResizeMergeSpecBuilder().
+		ForEntityType(proto.EntityDTO_CONTAINER).
+		ForCommodity(proto.CommodityDTO_VCPU).
```

Comment:
Maybe I am missing something since I am sure you tested it, what about the request commodity?

Reply:
This list can be extended, i only used the two commodities during development

Reply:
@pallavidn, I however get the request commodity actions merged even without adding them here. I was hoping to get some insight from you.
![image](https://user-images.githubusercontent.com/10027921/87738379-93a82380-c820-11ea-827d-f32ad8b4271e.png)


I have updated the PR anyways.